ivy/container/searching.py

```diff
@@ -424,6 +424,40 @@ def argwhere(
         -------
         ret
             Indices for where the boolean array is True.
+
+        Instance Method Examples
+        ------------------------
+
+        Using :class:`ivy.Array` instance method:
```

Comment:
Just need to move this example to ivy/array/searching.py under the argwhere method there.

Reply:
Moved this example to ivy/array/searching.py